src/Features/VisualBasic/Portable/ExtractMethod/VisualBasicMethodExtractor.VisualBasicCodeGenerator.vb

```diff
@@ -158,12 +149,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExtractMethod
                     Return declStatement.Parent
                 End Function
 
+                Private Function IsUnderModuleBlock() As Boolean
```

Comment:
vb specific helpers that were on VBSelectionResult, like with C#, these were called in one lcation.  so i just moved the helpers right to that location.